docs/graphkb/loading_data.md

```diff
@@ -180,25 +201,15 @@ Most popular resources which have pre-built loaders provided for GraphKB are lis
     As this is not an open data resource, using this loader will require licensing specific to your
     user/instance. This is an external knowledge base which can be imported as statements into GraphKB.
```

Comment:
knowledge base -> knowledgebase

Reply:
I looked this up at one point and it is supposed to be 2 words :shrug: